addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -177,7 +177,7 @@ Resources:
               StringEquals:
                 sts:ExternalId: !Ref ExternalId
       ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/AWSServiceCatalogAdminFullAccess
```

Comment:
I prefer to contribute a PR to upstream for this change. Then merge it back to this fork.